test/e2e/issuer.ts

```diff
@@ -128,7 +158,103 @@ export async function testE2E(issuerName: string, mTLS?: MTLSConfiguration): Pro
 	const token = await client.finalize(tokenResponse);
 
 	return (
-		(await verifyToken(BlindRSAMode.PSS, token, issuerPublicKey)) &&
+		(await origin.verify(token, issuerPublicKey)) &&
 		response.headers.get('Content-Type') === MediaType.PRIVATE_TOKEN_RESPONSE
 	);
 }
+
+async function testArbitraryBatchedRequest(issuerName: string, nTokens: number, mTLS?: MTLSConfiguration) {
+	// Create an array of per-token client instances.
+	const clients: publicVerif.Client[] = [];
```

Comment:
```suggestion
	const clients = new Array<publicVerif.Client>()
```

Reply:
what's the difference?

Reply:
the second one let's typescript do the type inference.

this was a suggestion mostly because this is the formatting used later on